user/favorite/store.go

```diff
@@ -148,6 +166,8 @@ func (db *DB) FindAll(ctx context.Context, userID string, filter []assignment.Ta
 			switch f {
 			case assignment.TargetTypeService:
 				allowServices = true
+			case assignment.TargetTypeRotation:
+				allowServices = true
```

Comment:
If we ask for rotations we shouldn't return services.

A second bool should be added, `allowRotations`, instead and the `findAll` query updated.